bindings/python/tests/test_triplestore.py

```diff
@@ -1,19 +1,71 @@
 # Skip test if rdflib is not available
+from pathlib import Path
+
 try:
     import rdflib
 except ImportError:
     import sys
-    sys.exit(44)
+    sys.exit(44)  # skip this test
+
+try:
```

Comment:
What is the advantage of this construction rather that just requiring pytest to be available upon running tests?

Reply:
Only to avoid a hard dependency on pytest. But I am not sure if it best practices though...

Writing the test as a script instead as a function as pytest requires, gives me the full local scope under my hands if during development by running the test via `ipython -i test_triplestore.py`. That is very convenient for testing different solutions and checking documentation.

Hence, I do not use pytest in general, except for pytest.raises(), which I find very nice and convenient for testing exceptions.